samples/client/others/typescript-angular-deep-object/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { TestBed } from '@angular/core/testing';
+import { AppComponent } from './app.component';
+import { provideHttpClient } from '@angular/common/http';
+import { HttpTestingController, provideHttpClientTesting } from '@angular/common/http/testing';
+
+describe('AppComponent', () => {
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [AppComponent],
+      providers: [
+        // ... other test providers
+        provideHttpClient(),
+        provideHttpClientTesting(),
+      ],
+    }).compileComponents();
+  });
+
+  it('should create the app', async () => {
```

Comment:
please add a test similar to https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/typescript-angular-v16-provided-in-root/tests/default/src/test/api.spec.ts where you make assertions about the deepObject request

Reply:
Done, see 71629f8d9a